concourse/scripts/publish.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -eou
+
+cd ./fauna-python-repository
+
+pip install -r requirements.txt
+
+PACKAGE_VERSION=$(python setup.py --version)
+echo "Going to publish python package: ${PACKAGE_VERSION}"
+
+pip install twine
+
+twine check dist/*
+twine upload dist/*
+
+echo "fauna-python@$PACKAGE_VERSION has been released" > ../slack-message/publish
```

Comment:
I know you'll care, so adding this. ;)
```suggestion
echo "fauna-python@$PACKAGE_VERSION has been released" > ../slack-message/publish

```